src/main/java/com/softserve/bookstore/repositories/UserRepository.java

```diff
@@ -95,6 +93,19 @@ public void addUsers(List<User> users) throws SQLException, UserNotFoundExceptio
         }
     }
 
+    @Transactional
+    public User addUser(User user) throws SQLException {
+        PreparedStatement userStatement = connection.prepareStatement(INSERT_USER);
+        userStatement.setString(1, user.getEmail());
+        userStatement.setString(2, user.getPassword());
+        userStatement.executeUpdate();
+        User lastUserAdded = findLastUsersAdded(1).get(0);
```

Comment:
as getting updated user:
 I think this is no the best way to do this, I mean getting first row in table after update
  issue # 1 : what if in parallell there will be another transaction which updates the table with new user, it might be possible that this findLastUsersAdded returns different user, which was updated in separate transaction, but this might be differ depending on isolation levels
  issue # 2 : might not be the best optimized SQL query
 I recommend you to use Id which you can probably get from executeUpdate method call, and using that id fetch newly added user

Reply:
I thought maybe it is not the best way, but I did't know how else to get the id from the database.
I find out that I can use getGeneratedKeys() method, so I made the changes and now I'm using it to get the id for the added user.